metpy/plots/skewt.py

```diff
@@ -621,6 +622,8 @@ def shade_cape(self, p, t, t_parcel, **kwargs):
         :func:`matplotlib.axes.Axes.fill_betweenx`
 
         """
+        if float(matplotlib.__version__[:3]) >= 2.1:
```

Comment:
No need to convert to float--string comparisons work:
```python
if matplotlib.__version__[:3] > '2.1':
```

Reply:
Also, couldn't we get away with just adding this as the default in `shade_area`?

Reply:
I just saw the top comment. I think it might make sense to put it in `shade_area` as the default, but allow users to override.

Reply:
I had duplicated that from `_mpl` - should we cleanup there?

Reply:
Actually scratch that:
` TypeError: '>=' not supported between instances of 'str' and 'float'`